docs/tutorials/downgrade-extension.md

```diff
@@ -102,34 +103,38 @@ Once the CRD safety checks are disabled and upgrade constraints are set, you can
 
 1. **Edit the ClusterExtension Resource:**
 
-   Modify the `ClusterExtension` custom resource to specify the target version and adjust the upgrade constraints.
+    Modify the `ClusterExtension` custom resource to specify the target version and adjust the upgrade constraints.
```

Comment:
Is this an accidental space?

Reply:
@trgeiger there were 3 spaces instead of 4 in this doc. Because of this numbered list was rendering incorrectly.

https://operator-framework.github.io/operator-controller/tutorials/downgrade-extension/
https://deploy-preview-1443--olmv1.netlify.app/tutorials/downgrade-extension/

Reply:
This was resolved in another PR: #1446 

Resolving